### PR TITLE
Do not create pending submission on bulk updates test

### PR DESCRIPTION
**Pull request by aaclan-ebi**
ebi-ait/dcp-ingest-central#552

### CODE REVIEW

**Review by jacobwindsor — APPROVED** [review]

File: tests/ingest_agents.py [hunk 1]
```diff
@@ -21,14 +21,18 @@ def __init__(self, deployment):
...
+    def upload(self, metadata_spreadsheet_path, is_update=False, project_uuid=None, submission_uuid=None):
         url = self.ingest_broker_url + '/api_upload'
         if is_update:
             url = self.ingest_broker_url + '/api_upload_update'
 
         data = {}
         if project_uuid:
             data['projectUuid'] = project_uuid
+
+        if submission_uuid:
```
Comment (ke4) [inline tests/ingest_agents.py:33]: What happens if `submission_uuid` is `None`?

File: tests/runners/bulk_update_runner.py [hunk 2]
```diff
@@ -100,12 +101,16 @@ def __upload_spreadsheet_and_create_submission(self, dataset_fixture, project_uu
         self.submission_envelope = self.ingest_api.envelope(self.submission_id)
 
     def __upload_modified_spreadsheet(self, path_to_spreadsheet):
-        self.ingest_broker.upload(path_to_spreadsheet, is_update=True)
+        submission_uuid = self.submission_envelope.uuid
+        project_uuid = self.project_uuid
+        self.ingest_broker.upload(path_to_spreadsheet, is_update=True, project_uuid=project_uuid,
```
Comment (ke4) [inline tests/runners/bulk_update_runner.py:106]: Could it be a case when the above `UUIDs` are `None`?

**Review by ke4 — COMMENTED** [review]
I added 2 questions. Thanks

File: tests/ingest_agents.py (hunk 1 above)
Comment (prabh-t) [reply tests/ingest_agents.py:33]: Just not passed in the post data. Case seems to be covered in broker SpreadsheetUploadService.async_upload (project and sub uuid can default to None).

**Review by prabh-t — APPROVED** [review]

File: tests/ingest_agents.py (hunk 1 above)
Comment (aaclan-ebi) [reply tests/ingest_agents.py:33]: Yeah, but if the submission UUID is none, it's going to create a new submission, it shouldn't for updates.

File: tests/runners/bulk_update_runner.py (hunk 2 above)
Comment (aaclan-ebi) [reply tests/runners/bulk_update_runner.py:106]: Yup, and it should be okay if both are none. But they have values in this case.

**Pull request merged**